assessments/homework.sql

```diff
@@ -93,7 +94,8 @@ INSERT INTO question_score_logs
 UPDATE instance_questions AS iq
 SET
     points_in_grading = least(iq.points + iq.current_value, aq.max_points) - iq.points,
-    score_perc_in_grading = least(iq.points + iq.current_value, aq.max_points) / aq.max_points * 100 - iq.score_perc
+    score_perc_in_grading = least(iq.points + iq.current_value, aq.max_points)
```

Comment:
Shouldn't this be `iq.points_in_grading` anyway, rather than `iq.points`? As in, this was an existing bug. We don't actually display `*_in_grading` anywhere yet, so it's escaped notice until now.